src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -207,6 +210,10 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
       if (this.isRoleStaff) {
         this.snackbarText = 'Business doesn\'t exist in LEAR.'
         this.snackbar = true
+
+        // Hide spinner.
+        this.$root.$emit('showSpinner', false)
```

Comment:
I don't like the duplicated "hide spinner" code for early exits from this method. Maybe this can be refactored out when some of this functionality is moved to the mixin or store ...?

Reply:
PS What do you think of the spinner? It came in really handy today when the APIs were slow 😆

Reply:
I love it Sev. I've also experienced slowness yesterday so I think it's super handy.